### PR TITLE
Distinguish current page in Sidebar,disable clicking of the active link and prettify the URL Fixes #22

### LINKED ISSUE #22: The link to the current page in the sidebar should be distinguished, and non-clickable.
It's hard to tell where I currently am from looking at the sidebar. It would be better if the link to the page I'm currently on is highlighted somehow, and also made non-clickable.

### PR DESCRIPTION
**Pull request by makoscafee**
This wont't affect links which starts with #. It is smart enough to remove the # in the link.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -91,4 +92,7 @@ oppiaGithubPages.run(['$location', '$rootScope', function($location, $rootScope)
     $rootScope.projectTag = current.$$route.projectTag;
     $rootScope.title = 'Oppia: ' + current.$$route.projectTag;
   });
+  $rootScope.isActive = function(route) {
+    return route === $location.path();
+  }
```
Comment (seanlip) [inline app.js]: add semicolon

File: app.js [hunk 2]
```diff
@@ -84,6 +83,8 @@ oppiaGithubPages.config(['$routeProvider', function($routeProvider) {
     .otherwise({
       redirectTo: '/'
     });
+     //Use the HTML5 History API to prettfy URL(by removing #)
```
Comment (seanlip) [inline app.js]: typo: prettify
nit: add space before '('

(also, nice addition! That # was annoying)

File: pages/sidebar.html [hunk 3]
```diff
@@ -1,32 +1,30 @@
...
     <hr style="margin: 20px 0;">
-
-    <li><a href="#Parameters">Reference: Parameters</a></li>
-    <li><a href="#DesignTips">Reference: Design Tips</a></li>
-    <li><a href="#EmbeddingAnExploration">Reference: Embedding</a></li>
-    <li><a href="https://github.com/oppia/oppia/wiki">Developer wiki</a></li>
+    <li><a ng-class="{active:isActive('/Parameters')}" href="/Parameters">Reference: Parameters</a></li>
+    <li><a ng-class="{active:isActive('/DesignTips')}" href="/DesignTips">Reference: Design Tips</a></li>
+    <li><a ng-class="{active:isActive('/EmbeddingAnExploration')}" href="/EmbeddingAnExploration">Reference: Embedding</a></li>
+    <li><a ng-class="{active:isActive('https://github.com/oppia/oppia/wiki')}" href="https://github.com/oppia/oppia/wiki">Developer wiki</a></li>
```
Comment (seanlip) [inline pages/sidebar.html]: I don't think this one needs an 'active' ng-class; it's an external link

File: pages/sidebar.html [hunk 4]
```diff
@@ -1,32 +1,30 @@
...
 <div id="sidebar">
   <ul class="primary">
-    <li><a href="#"><strong>Home</strong></a></li>
-    <li><a href="#Screenshots">Screenshots</a></li>
-    <li><a href="#WhatIsOppia">What is Oppia?</a></li>
-    <li><a href="#KeyConceptsInOppia">Key concepts in Oppia</a></li>
-    <li><a href="#TheExplorationGallery">The exploration gallery</a></li>
-    <li><a href="#TheLearnerView">The learner view</a></li>
-
+    <li><a ng-class="{active:isActive('/')}"href="/">Home</a></li>
```
Comment (seanlip) [inline pages/sidebar.html]: nit: can you add a space after the colon? Same for the rest below.

File: static/css/sidebar.css [hunk 5]
```diff
@@ -5,7 +5,8 @@
   margin: 0 auto;
 }
 
-#sidebar a:hover {
+#sidebar a:hover,
+#sidebar a:focus{
```
Comment (seanlip) [inline static/css/sidebar.css]: nit: add space before {

File: static/css/sidebar.css [hunk 6]
```diff
@@ -32,3 +27,11 @@
 #sidebar>.primary>li>a {
   height: 16px
 }
+
+.active {
+  font-weight: 600;
```
Comment (seanlip) [inline static/css/sidebar.css]: nit: alphabetize css attrs

File: app.js (hunk 1 above)
Comment (makoscafee) [inline app.js]: Done

File: app.js (hunk 2 above)
Comment (makoscafee) [inline app.js]: Done

File: pages/sidebar.html (hunk 3 above)
Comment (makoscafee) [inline pages/sidebar.html]: I removed the active ng-class for that link

File: pages/sidebar.html (hunk 4 above)
Comment (makoscafee) [inline pages/sidebar.html]: Done

File: static/css/sidebar.css (hunk 5 above)
Comment (makoscafee) [inline static/css/sidebar.css]: Done

File: static/css/sidebar.css (hunk 6 above)
Comment (makoscafee) [inline static/css/sidebar.css]: Done

File: app.js [hunk 7]
```diff
@@ -14,11 +14,10 @@
 
 var oppiaGithubPages = angular.module('oppiaGithubPages', ['ngRoute']);
 
-oppiaGithubPages.config(['$routeProvider', function($routeProvider) {
+oppiaGithubPages.config(['$routeProvider','$locationProvider', function($routeProvider,$locationProvider) {
```
Comment (seanlip) [inline app.js]: Sorry, missed this nit -- in both cases, add single space after ','

Comment (makoscafee) [inline app.js]: Done

**Pull request merged**